app/assets/scripts/views/ThreeW/project-form.js

```diff
@@ -509,6 +509,11 @@ class ProjectForm extends React.PureComponent {
         </InputSection>
 
         <footer>
+          <button
```

Comment:
This seems like a reasonable hack :-) - can we add a comment in the code as to why this hidden button is here? Maybe difficult to figure out without context later.

Reply:
Sure, will do